dbus-codegen/src/main.rs

```diff
@@ -47,15 +47,22 @@ Defaults to 'RefClosure'."))
              .help("If present, skips a specific prefix for interface names, e g 'org.freedesktop.DBus.'."))
         .arg(clap::Arg::with_name("futures").short("f").long("futures")
              .help("Generates code to use with futures 0.3 (experimental)"))
+        .arg(clap::Arg::with_name("file").required(false).help("D-Bus XML Introspection file"))
         .get_matches();
 
+    if matches.value_of("destination").is_some() && matches.value_of("file").is_some() {
```

Comment:
You should make use of https://kbknapp.github.io/clap-rs/clap/struct.ArgMatches.html#method.is_present here.

Reply:
I'm new to rust. tank you for the link. I have updated the MR

Reply:
I'm not very experienced either and with each day, I forget more. :( But together I hope we can tame the beast. :) Thanks, I'll test it.